### PR TITLE
Get Available Markets

### PR DESCRIPTION
**Pull request by eleonora2687**

### CODE REVIEW

File: src/main/java/spotify/models/markets/MarketFull.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package spotify.models.markets;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MarketFull {
+
+    private List<String> markets = new ArrayList<String>() ;
+    public List<String> getAllMarkets() {
```
Comment (jzheng2017) [inline src/main/java/spotify/models/markets/MarketFull.java:9]: I prefer `getMarkets`

File: src/main/java/spotify/api/impl/MarketApiRetrofit.java
Comment (jzheng2017) [inline src/main/java/spotify/api/impl/MarketApiRetrofit.java:1]: Fix the whitespace formatting please.

**Comment by jzheng2017** [comment]
Looks good overall but please fix the whitespace formatting in general for all the files. In addition to that write unit tests for the added functionality. See the existing tests for reference.

File: src/main/java/spotify/models/markets/MarketFull.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package spotify.models.markets;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MarketFull {
+
+    private List<String> markets = new ArrayList<String>() ;
```
Comment (jzheng2017) [inline src/main/java/spotify/models/markets/MarketFull.java:8]: Remove the ArrayList instantiation

**Review by jzheng2017 — CHANGES_REQUESTED** [review]

**Comment by eleonora2687** [comment]
Can you please check the changes i made (whenever you get the time)  and let me know what else should i do next?
i've put A LOT of effort to make all this(the project is big and it it very easy to "get lost" here trying to adapt things here 😓

Thanks in advance

**Comment by jzheng2017** [comment]
> Can you please check the changes i made (whenever you get the time) and let me know what else should i do next? i've put A LOT of effort to make all this(the project is big and it it very easy to "get lost" here trying to adapt things here 😓
> 
> Thanks in advance

Thanks. I've been a bit busy so I haven't been able to check. I will review the PR soon. Thanks for the patience.

**Comment by jzheng2017** [comment]
PR looks great with no additional comments. Approved for now. I will test it soon and if it works fine then it can be merged.

**Comment by jzheng2017** [comment]
I have tested this PR and it works without any issue. It's okay for merge. Thank you for your work.

**Pull request merged**